setup.py

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from sugar.activity import bundlebuilder
+from sugar.3activity import bundlebuilder
```

Comment:
No such module `sugar.3activity`.  Add _test setup.py_ to your checklist for future ports.

Reply:
Apologies. Such mistakes won't repeat.